var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -107,28 +107,23 @@ def install(self, spec, prefix):
             make("install")
             self.install_test()
 
-    def setup_dependent_package(self, module, dspec):
-        # libsatlas.[so,dylib,dll ] contains all serial APIs (serial lapack,
-        # serial BLAS), and all ATLAS symbols needed to support them. Whereas
-        # libtatlas.[so,dylib,dll ] is parallel (multithreaded) version.
-        name = 'libsatlas.%s' % dso_suffix
-        libdir = find_library_path(name,
-                                   self.prefix.lib64,
-                                   self.prefix.lib)
-
-        if '+shared' in self.spec:
-            self.spec.blas_shared_lib   = join_path(libdir, name)
-            self.spec.lapack_shared_lib = self.spec.blas_shared_lib
+    @property
+    def blas_libs(self):
+        shared = True if '+shared' in self.spec else False
+        return find_libraries(['libsatlas'], root=self.prefix, shared=shared, recurse=True)
```

Comment:
btw, the threaded variant should **NOT** be called `openmp`, atlas by default uses`pthreads`, but can be forced to use `openmp` http://math-atlas.sourceforge.net/errata.html:

> Building using OpenMP: In general, this is a bad idea, since OpenMP tends to much slower than pthreads for ATLAS use. However, if your own application uses OpenMP, sometimes pthread usage can slow down your own threads, making it worthwhile to damage ATLAS performance in order to improve your OpenMP performance. In ATLAS, thread affinity is the main reason pthreads wins against OpenMP, and so OS~X (and probably FreeBSD), which don't support real affinity, are the platforms where it makes sense to use OpenMP regardless. To force ATLAS to use OpenMP rather than pthreads, you must add the following flags to your configure line:
   -Si omp 1 -F alg -fopenmp